docs/Guide/application-commands/application-command-registry/registering-context-commands.mdx

```diff
@@ -200,11 +146,18 @@ export class SlashCommand extends Command {
 }
 ```
 
-[aaacr]: ./acquiring-application-command-registry.mdx
-[acr]: ../../../Documentation/api-framework/classes/ApplicationCommandRegistry
+:::info
+
+Since `contextMenuRun` runs both kind of context commands, we recommend using either `isMessageContextMenu()` or
+`isUserContextMenu()` to ensure the correct kind of context menu application command is executing your command.
+
+:::
```

Comment:
can be reverted after the other reverts

Reply:
I think we should keep it to inform readers that `isMessageContextMenu()` exists